src/naksu/box/box.go

```diff
@@ -64,7 +64,7 @@ func getVBoxManageOutput() string {
 	}
 
 	cacheShowVMInfo.updateStarted = time.Now().Unix()
-	return mebroutines.RunVBoxManage([]string{"showvminfo", "--machinereadable", boxID})
+	return mebroutines.RunVBoxManage([]string{"showvminfo", "--machinereadable", boxID}, false)
```

Comment:
En ole ehtinyt vielä testailla tätä mitenkään, mutta minulle jää hiukan hämäräksi, voidaanko näistä tapauksista palautua jotenkin järkevästi. Eli siis saadaanko hyötyä siitä, että naksu ei välittömästi lopeta suoritusta virheen sattuessa?

Reply:
Yleisesti näissä kutsuissa haetaan jotain virtuaalikoneen tietoja, jonka perusteella sitten tehdään asioita. Jos arvoja ei saada, palautetaan tyhjiä arvoja ja kutsuva pää sitten 

Tässä onkin oikeastaan koko commitin pihvi: On selkeämpää käynnistää Naksu ja näyttää palvelimen tilanne (esim. se, että versiota ei voida näyttää) kuin sulkea ohjelma ykskantaan. Näin käyttäjällä on mahdollisuus esim. klikata "Poista palvelin", jolla voi korjata useimmat solmut.

Reply:
Totta! Helposti unohtuu tuo palvelimen poistotoiminto, kun useammin käytetään "Poista kokeet" (joka on vagrant destroy, joka ei tietenkään näissä tapauksissa toimi).